gandalf.js

```diff
@@ -1,96 +1,90 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!